src/components/data/DataTable.js

```diff
@@ -255,6 +257,38 @@ const DataTable = props => {
       }, [
         childrenBefore && childrenBefore({ entities, columnSettings, showColumnSettingsModal }),
         div({ style: { flexGrow: 1 } }),
+        isSearchAwesomeNow && h(MenuTrigger, {
+          side: 'bottom',
+          closeOnClick: false,
+          popupProps: { style: { width: 250 } },
+          content: h(Fragment, [
+            div({ style: { padding: '1rem' } }, [
+              div({ style: { fontWeight: 600 } }, ['Search logic']),
+              div({ role: 'radiogroup', 'aria-label': 'choose an operator to use for advanced search' }, [
```

Comment:
Alternatively, since there's already a visible label ("Search logic") for this group, `aria-labelledby` could be used to to associate it with this group vs adding a separate label with `aria-label`.

https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-labelledby